opentelekomcloud/acceptance/ecs/utils.go

```diff
@@ -68,6 +68,36 @@ func getFlavors() (map[string][]*quotas.ExpectedQuota, error) {
 	return resultsQ, nil
 }
 
+func getFlavorName() string {
+
+	resultsQ, err := getFlavors()
+	if err != nil {
+		panic("failed to get server flavors")
```

Comment:
panic?

Reply:
yes, because the function uses the same request as in the init function. And it will be unexpected error if it can't get the flavors.

Reply:
`init` function expects error, not panic. How do you handle it?